src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -37,26 +37,37 @@ def create_rss_notifications(event)
   end
 
   def notification_dynamic_params(event)
-    if event.eventtype == 'RequestStatechange'
+    if event.eventtype == 'Event::RequestStatechange'
       return {
         notifiable_type: 'BsRequest',
```

Comment:
You can extract this return block and the others below into methods. Doing so you'll reduce the method line numbers and fix the Code Climate issues

Reply:
We refactored the code. Now Code Climate is passing. Please have a look again.